src/base64Utils.js

```diff
@@ -13,24 +17,29 @@ function intArrayFromBase64(s) {
 #if ENVIRONMENT_MAY_BE_NODE
   if (typeof ENVIRONMENT_IS_NODE != 'undefined' && ENVIRONMENT_IS_NODE) {
     var buf = Buffer.from(s, 'base64');
-    return new Uint8Array(buf.buffer, buf.byteOffset, buf.length);
+#if GZIP_EMBEDDINGS
+  return new Response(new Blob([buf]).stream().pipeThrough(new DecompressionStream("gzip"))).arrayBuffer();
```

Comment:
Can you move the `DecompressionStream` from here and do it instead in `tryParseAsDataURI` (its only caller).

Then we don't need to duplicate the logic, and it keeps `intArrayFromBase64` more true to its name.